### DISCUSSION TITLE
How to have the app.HASH.js being automatically referenced in the index.html?

### DISCUSSION THREAD

**Discussion by ClemRz** [Day8/re-frame-template#151] [Q&A]
Hi!

I'm quite new using re-frame so please pardon me if my question is either stupid or already answered.

I added the flag `:module-hash-names true` in the build of my app so I don't have to invalidate caches when deploying.

The thing is that in the `resources/public/index.html`, the `app.js` is referenced statically. When I open it there is a `GET .../public/js/compiled/app.js net::ERR_FILE_NOT_FOUND` exception in the browser's console.

https://github.com/day8/re-frame-template/blob/594e17b1daf559fbb289abeba1f20988b390825d/src/leiningen/new/re_frame/resources/public/index.html#L20

Please can anyone help me understand what step am I missing in the release process?

**Comment by ClemRz** [comment]
I think I found a solution for this: https://clojurians-log.clojureverse.org/shadow-cljs/2018-03-23

Too bad it is not better integrated into the framework.

**Reply by superstructor** [reply]
@ClemRz PR welcome.
